lib/modules/datasource/rubygems/versions-datasource.ts

```diff
@@ -110,14 +150,14 @@ export class RubyGemsOrgDatasource extends Datasource {
     }
   }
 
-  private static isDataStale(): boolean {
-    return getElapsedMinutes(lastSync) >= 5;
+  private isDataStale(): boolean {
+    return getElapsedMinutes(this.registryCache.lastSync) >= 15;
   }
 
   private updateRubyGemsVersionsPromise: Promise<void> | null = null;
 
   async syncVersions(): Promise<void> {
-    if (RubyGemsOrgDatasource.isDataStale()) {
+    if (this.isDataStale()) {
       this.updateRubyGemsVersionsPromise =
         this.updateRubyGemsVersionsPromise ?? this.updateRubyGemsVersions();
```

Comment:
@rarkins - I believe that this line (pre-existing in the get-rubygems-org that this was based on) deals with the race condition. It will only fire a new request if there is not one already in flight.